src/cufinufft.cu

```diff
@@ -85,7 +85,7 @@ int CUFINUFFT_MAKEPLAN(int type, int dim, int *nmodes, int iflag,
 	nmodes  a size 3 integer array, nmodes[d] is the number of modes in d
 	        dimension
 	iflag   if >=0, uses + sign in exponential, otherwise - sign (int)
-    ntransf number of transforms performed in exec stage
+                ntransf number of transforms performed in exec stage
```

Comment:
This should probably be reverted? Maybe the GH display is playing tricks with the whitespace. (Also below.)

Reply:
o.O --- ah! i think i know what happened: my editor automagically does tab->space and maybe my ts was set to 8 (another project has that)